src/foam/nanos/ndiff/services.jrl

```diff
@@ -0,0 +1,22 @@
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"ndiffDAO",
+  "serve":true,
+  "authenticate":false,
+  "lazy":false,
+  "serviceScript": """
+    dao  = new foam.dao.EasyDAO.Builder(x)
+      .setOf(foam.nanos.ndiff.NDiff.getOwnClassInfo())
+      .setDecorator(new foam.nanos.ndiff.NDiffDAO(x, new foam.dao.NullDAO(x, foam.nanos.ndiff.NDiff.getOwnClassInfo())))
```

Comment:
Ok, couple of options
1. this is not required. and NDiffDAO created in JDAO looksup 'ndiffDAO' as needed. 
2. the composite dao just setups two daos    x.get("ndiffDAO"), and the original delegate. and puts to each.  This decorator here will handle the put on the ndiffdao and do the appropriate adapting, and doesn't have to reply or delegate itself.

Reply:
the JRL file has changed; we now grab the ndiffDAO service in the NDiffDAO decorator.